yt/frontends/cholla/data_structures.py

```diff
@@ -103,7 +103,9 @@ def _parse_parameter_file(self):
             attrs = h5f.attrs
             self.parameters = dict(attrs.items())
             self.domain_left_edge = attrs["bounds"][:].astype("=f8")
-            self.domain_right_edge = self.domain_left_edge + attrs["domain"][:].astype("=f8")
+            self.domain_right_edge = self.domain_left_edge + attrs["domain"][:].astype(
+                "=f8"
+            )
```

Comment:
This is one of the rare cases where black makes things objectively worse looking... here's a possible alternative
```suggestion
            dre = attrs["bounds"][:] + attrs["domain"][:]
            self.domain_right_edge = dre.astype("=f8")
```